os/pm/Kconfig

```diff
@@ -70,8 +60,8 @@ config PM_TIMEDWAKEUP
 if PM_TIMEDWAKEUP
 
 config PM_MIN_SLEEP_TIME
```

Comment:
I feel the naming of this config is not correct. MIN_SLEEP_TIME seems to indicate minimum time pm can sleep or something else. Can we change this config to PM_SLEEP_ENTRY_WAIT_MS or something else?

Reply:
Done